_episodes/07-conflicts.md

```diff
@@ -33,18 +33,10 @@ There are several ways to do that as we will see.
 - Create two branches from `master`.
 - On the two branches make **different modifications** to the amount of the **same ingredient**.
 
-These are my modifications:
+To view the modifications:
 
 ```shell
-$ git show-branch
-
-* [dislike-cilantro] please no cilantro
- ! [like-cilantro] more cilantro please
-  ! [master] Merge branch 'experiment' into less-salt
----
-*   [dislike-cilantro] please no cilantro
- +  [like-cilantro] more cilantro please
---- [master] Merge branch 'experiment' into less-salt
+$ git graph
```

Comment:
Here we should indicate that we rely on an alias. If somebody missed the alias part and try this, it will fail for them.

Reply:
Yes, that us true. I will include the full command as this is defined in another lesson. May be better to avoid too much dependencies on other lessons, just in case this get ported to somewhere else.

Reply:
It is OK to use it but add a line or two that gives people chance to catch up who missed the definition of the alias.